bot/cogs/spookyavatar.py

```diff
@@ -0,0 +1,50 @@
+import aiohttp
+from io import BytesIO
+
+from discord.ext import commands
```

Comment:
The correct PEP8 order is:

```python
from io import BytesIO

import aiohttp
import discord
from discord.ext import commands
from PIL import Image

from bot.resources import spookifications
```

`flake8` should tell you this

Reply:
`flake8` doesn't seem to complain either way, but I've reordered the imports as you asked.

Reply:
```
(hacktoberbot) ~/D/G/h/bot> flake8 ./spookyavatar.py
./spookyavatar.py:2:1: I100 Import statements are in the wrong order. 'from io import BytesIO' should be before 'import aiohttp' and in a different group.
./spookyavatar.py:2:1: I201 Missing newline between import groups. 'from io import BytesIO' is identified as Stdlib and 'import aiohttp' is identifiedas Third Party.
./spookyavatar.py:5:1: I100 Import statements are in the wrong order. 'import discord' should be before 'from discord.ext import commands'
```